src/structure_dump.cpp

```diff
@@ -542,35 +555,49 @@ void structure::load_chunk_layout(const std::vector<grid_volume> &gvs,
   check_chunks();
 }
 
-void structure::load(const char *filename) {
-  h5file file(filename, h5file::READONLY, true);
+void structure::load(const char *filename, bool single_parallel_file) {
+  h5file file(filename, h5file::READONLY, single_parallel_file, !single_parallel_file);
 
-  if (verbosity > 0) master_printf("reading epsilon from file \"%s\"...\n", filename);
+  if (verbosity > 0) printf("reading epsilon from file \"%s\" (%d)...\n", filename, single_parallel_file);
```

Comment:
Again, it seems `printf` should be `master_printf`.

Reply:
ditto. Please LMK if that is not advisable for some reason.